src/main/java/com/thoughtworks/aceleradora/dominio/GerenciadorNivel.java

```diff
@@ -0,0 +1,116 @@
+package com.thoughtworks.aceleradora.dominio;
+
+import java.util.Scanner;
+
+public class GerenciadorNivel {
+
+    private Scanner entrada = new Scanner(System.in);
+    private Diagnostico novoDiagnostico;
+
+    public GerenciadorNivel(Diagnostico novoDiagnostico) {
```

Comment:
Assim como o diagnóstico, acho que o Scanner também poderia ser passado por construtor, assim podemos aproveitar o Scanner criado lá no método main. Isso também ajudaria a diminuir o acoplamento entre `Scanner` e `GerenciadorNivel` no momento em que esse desacoplamento for necessário.